src/lib/apis/dashboard/orders-status.api.ts

```
@@ -0,0 +1,20 @@
+export async function fetchStatistics(): Promise<StatisticsResponse> {
+    // Make GET request to our Next.js API route
+    const res = await fetch("/api/statistics", {
+        method: "GET",
+    });
+
+    // Parse JSON payload from response
+    const payload = await res.json();
+
+    // Check if response is not OK, then log error and throw
+    if (!res.ok) {
+        console.error("Failed to fetch statistics:", payload);
```

ESLint

[error] Unexpected console statement.

(no-console)